test/cookbooks/test/recipes/extension.rb

```diff
@@ -2,11 +2,11 @@
 
 postgresql_server_install 'package' do
   action [:install, :create]
-  initdb_locale 'en_US.utf8'
+  initdb_locale 'en_US.UTF-8'
```

Comment:
Is this actually what CircleCi/dokken expects? I put it lower case because no one can decide on which case to use... (and it *was* working as is)

Reply:
I tried to converge it with testkitchen locally and it fails with locale 'en_US.utf8'.

Reply:
It's probably a dokken vs kitchen thing and who built the images.

Reply:
OK, so removing?

Reply:
can you push another commit please? I just poked CircleCI to run the tests on forked PRs :)

Reply:
pushed